BUILD.md

```diff
@@ -232,7 +237,7 @@ Same as for [macOS](#3-compile-and-create-package)
     https://wxwidgets.org/
 [NSIS]:
     http://nsis.sourceforge.net/Download
-[CMake support in Visual Studio 2017]:
```

Comment:
Because it's not needed froma user-perspective view, he need only to open folder, not know what there is behind.